datashuttle/utils/ds_logger.py

```diff
@@ -0,0 +1,114 @@
+import logging
+from pathlib import Path
+from typing import Any, List, Optional
+
+import fancylog as package
+from fancylog import fancylog
+from rich import print as rich_print
+from rich.console import Console
+from rich.filesize import decimal
+from rich.markup import escape
+from rich.text import Text
+from rich.tree import Tree
+
+from . import utils
+
+
+def start(
+    path_to_log: Path, name: str, variables: Optional[List[Any]]
+) -> None:
+    """"""
+    fancylog.start_logging(
+        path_to_log,
+        package,
+        filename=name,
+        variables=variables,
+        verbose=False,
+        timestamp=True,
+        file_log_level="INFO",
+        write_git=False,
```

Comment:
Sorry to raise so many fancylog questions, but I know the package well... Any reason to not write this if it's available? If someone is using a version that's not released, it's useful to get the relevant git commit straight from the log. The info should be in the setuptools_scm version, but I think this is easier.

Reply:
Not at all very useful. I think I fundamentally misunderstood the purpose of the log. Previously I thought it would be mainly used to researchers to keep track of any operations they might have done and look through the history. Us having spoken further it is clear this is not the way logs are typically used and instead should contain as much information for debugging as possible. A few options were turned off to keep the log cleaner for the user, but this is the wrong approach. Will set `True`!

Reply:
I don't know if this is a general principle, but I use `logging.info()` sparingly to keep the console logs clean for the user, but then I send absolutely everything to the log file (with `logging.debug()` but also fancylog options) to aid (usually remote) debugging later.